src/Core/Constants.cs

```diff
@@ -41,6 +41,8 @@ public static class Constants
         public const string NotificationDataType = "Type";
         public const string PasswordlessLoginRequestKey = "passwordlessLoginRequest";
         public const string PreLoginEmailKey = "preLoginEmailKey";
+        public const string ConfigsKey = "configsKey";
+        public const string DisplayEuEnvironmentFlag = "display-eu-environment";
```

Comment:
⛏️ IMO this can be moved to the `ConfigResponse` given that this flag is specific to the dictionary inside. So when more feature flags are added we have all of the keys in the same place.